src/Razor/src/Microsoft.AspNetCore.Razor.Language/CodeGeneration/CodeRenderingContext.cs

```diff
@@ -40,5 +39,9 @@ public abstract class CodeRenderingContext
         public abstract void RenderChildren(IntermediateNode node);
 
         public abstract void RenderChildren(IntermediateNode node, IntermediateNodeWriter writer);
+
+        public virtual void AddLinePragma(LinePragma linePragma)
```

Comment:
Already spoke to @NTaylorMullen about this. This needs to be public because we share `CodeWriterExtensions.cs` which uses this method using compile include in `Mvc.Razor.Extensions`.

Reply:
seems fine.